### ISSUE TITLE
Pytorch error line=405 error=11 : invalid argument upsampling.py:129: 

### ISSUE THREAD

**Issue by segalinc** [ayooshkathuria/pytorch-yolo-v3#64]
Hello,
I am useing 4 gpu RTX 2080 cuda 10 and torch 1.0 and when I run the code, even if it's working and it's just a warning I get this message.

THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=405 error=11 : invalid argument
python3.6/site-packages/torch/nn/modules/upsampling.py:129: UserWarning: nn.Upsample is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.{} is deprecated. Use nn.functional.interpolate instead.".format(self.name))

When I run the face detector with other code then this warning turn into a real error. 
all I found was related to some problem with cuda 9 or torch version but I already have the correct ones.

Thank you

**Comment by segalinc** [comment]
Solved compiling pytorch froms ource if anyone else have the same problem

**Comment by Jason0409** [comment]
titan rtx cuda 10 and torch 1.0.1 same problem

**Comment by Jason0409** [comment]
I just check the status of GPU,but it sames the gpu still work.. 
THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=405 error=11 : invalid argument
Epoch 1, train loss: 1.0066, valid loss: 0.6847.
Validation loss decreased (inf --> 0.684684).  Saving model ...
Thu Mar 28 16:48:05 2019 Epoch: 2

**Comment by fourth-archive** [comment]
@Jason0409 @cristinasegalin highly recommend this repository for multiGPU and multithreaded PyTorch yolov3 training: https://github.com/ultralytics/yolov3

## Speed

https://cloud.google.com/deep-learning-vm/  
**Machine type:** n1-standard-8 (8 vCPUs, 30 GB memory)  
**CPU platform:** Intel Skylake  
**GPUs:** 1-4x P100 ($0.493/hr), 1-8x V100 ($0.803/hr)  
**HDD:** 100 GB SSD  
**Dataset:** COCO train 2014 

GPUs | `batch_size` | batch time | epoch time | epoch cost
--- |---| --- | --- | --- 
<i></i> |  (images)  | (s/batch) |  |
1 P100 | 16 | 0.39s  | 48min  | $0.39
2 P100 | 32 | 0.48s | 29min | $0.47
4 P100 | 64 | 0.65s | 20min | $0.65
1 V100 | 16 | 0.25s  | 31min | $0.41
2 V100 | 32 | 0.29s | 18min | $0.48
4 V100 | 64 | 0.41s | 13min | $0.70
8 V100 | 128 | 0.49s | 7min | $0.80
